docsets/PyMC3/docset.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "PyMC3",
+    "version": "3.11.2",
+    "archive": "Sample.tgz",
```

Comment:
I think this is supposed to point to `PyMC3.tgz`.

Reply:
Good catch!
Addressed in latest commit.